functions.php

```diff
@@ -120,11 +125,11 @@ function _s_widgets_init() {
  * Enqueue scripts and styles.
  */
 function _s_scripts() {
-	wp_enqueue_style( '_s-style', get_stylesheet_uri() );
+	wp_enqueue_style( '_s-style', get_stylesheet_uri(), array(), _S_VERSION );
 
-	wp_enqueue_script( '_s-navigation', get_template_directory_uri() . '/js/navigation.js', array(), '20151215', true );
+	wp_enqueue_script( '_s-navigation', get_template_directory_uri() . '/js/navigation.js', array(), _S_VERSION, true );
```

Comment:
Can we leave the version as is? The script does not get changed with every theme version. This way we would not need the constant.

Reply:
Except that the scripts have been changed a couple of times already and people keep forgetting to update this cache-busting number, so I would *strongly* recommend leaving it as is (with the constant), as refreshing the script once on every new release is a minor slow-down, while serving an out-of-date script can break a site.

Reply:
I don't agree but I don't want to stand in the way of this PR being merged. We might need a PR for https://github.com/Automattic/underscores.me/ too to search and replace `_S`.